src/schemata/definitions/info.yml

```diff
@@ -1,33 +1,35 @@
 ---
 type: object
-title: Info Section Schema
+title: Info
 description: |
-  The info section of a workflow or step template holds general information
-  about the template.
+  The info section of a Workflow (Step) Template holds general information about the template.
 required:
   - title
+  - uuid
+  - version
 properties:
   title:
     type: string
-    title: Title
+    maxLength: 200
```

Comment:
This field was added.

<details>
<summary>200 chars</summary>

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repel.

</details>